components/third_party/adblock/lists/peter_lowes/LICENSE

```diff
@@ -0,0 +1,3 @@
+its software.
```

Comment:
Is there something missing here?

Reply:
Perhaps I pressed "resolve" too quickly! :sweat_smile: 
https://github.com/brave/brave-core/pull/6256#discussion_r462595437